css/index.css

```diff
@@ -85,6 +115,14 @@ p {
     padding-bottom: 20px;  
 }
 
+.top-image {
+    margin-top: 40px;
+    margin-bottom: 40px;
```

Comment:
Not sure if this would cancel? 

being the same margins on each side, but could be other things influencing it if there was a net change.